apps/backoffice-v2/package.json

```diff
@@ -34,6 +34,7 @@
     "pnpm": ">=8.0.0"
   },
   "scripts": {
+    "spellcheck": "cspell \"*\"",
```

Comment:
`cspell "*"` will only check the current directory and nothing below. Did you intend to use `cspell .`?